src/app/login-signup/sign-up/sign-up.component.sass

```
@@ -6,6 +6,14 @@
     margin-bottom:              17.3333333333333px
     border:                     none
     border-bottom:              3px solid var(--theme1)
+    position:                   relative
+
```

Adding this will to fix to some degree the bouncing problem with checks and times.
```suggestion
    height:                   30px
```